src/components/Search.jsx

```diff
@@ -44,19 +44,19 @@ export default class Search extends Component {
 
   render() {
     return (
-      <div>
+      <div className='search-container'>
       <input
         id='searchInput'
         className='search-input'
         type='text' 
         placeholder='search'
         onChange={ this.handleChange.bind(this, 'search') } 
         onKeyDown={ this.handleKeyPress }/>
-        <button
+        <input
```

Comment:
Learned about the image input type which allows the ability to add an image source and is handled as a button. This made it very easy to add the search icon instead of making a button and putting the icon as the background image.